from_now/from_now.go

```diff
@@ -0,0 +1,112 @@
+package from_now
+
+import (
+	"fmt"
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type from_now struct{}
+
+func init() {
+	extpoints.Register("from-now", from_now{})
+}
+
+func (from_now) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (from_now) Summary() string {
+	return "Returns a timestamp which is DURATION ahead in the future."
+}
+
+func (from_now) Usage() string {
+	usage := "Usage: taskcluster from-now DURATION"
+	usage += "\n"
+	return usage
+}
+
+func (from_now) Execute(context extpoints.Context) bool {
+	duration := context.Arguments["DURATION"].(string)
+
+	offset, err := parseTime(duration)
+	
+	if err {
+		panic("String: '" + duration + "' isn't a time expression")
```

Comment:
The convention here seems to be to output to `os.Stderr` and `return false`. We tend to reserve `panic` for when things go really bad (a segfault, for example).

    fmt.Fprintf(os.Stderr, "Formatted error message")
    return false